doc/releases/0.98.0.rst

```diff
@@ -0,0 +1,201 @@
+.. _release0.98.0:
+
+SpikeInterface 0.98.0 release notes
+-----------------------------------
+
+7th July 2023
+
+
+Main changes:
+
+* Implemented motion correction in SpikeInterface
+* Get rid of to_dict in _kwargs and use JSON serializer
+* _ind to _index in complex dtype representation
+* Moved to src/spikeinterface structure
+* Reformat to black and use pre-commit CI
```

Comment:
I do not feel that black and pre-commit is somhow an improvement...
Promoting to *main changes* hurt me a lot.